lint/testdata/emptyFallthrough/negative_tests.go

```diff
@@ -2,26 +2,48 @@ package checker_tests
 
 import "fmt"
 
-func noWarningsNonEmptyFallthrough(i int) int {
+func noWarningsNonEmptyFallthrough(i int) bool {
```

Comment:
What was the point to change int -> bool?

Reply:
the point was to harmonize with the negative tests, where always this signature was used. so all tests look similar (it is only a stylistic change).

Reply:
Yeah, I thought it's stylistic, just wanted to ensure.
BTW I think we don't have any styleguides for tests 😉